entrypoint.sh

```diff
@@ -28,8 +24,8 @@ then
 fi
 
 export RAILS_RELATIVE_URL_ROOT=${APPLICATION_ROOT:-'/app/root'}
-export SCRIPT_NAME=${APPLICATION_ROOT}
+export SCRIPT_NAME=${RAILS_RELATIVE_URL_ROOT}
 
-echo '{"ts": "'"$(date -u +%FT%T.%3NZ)"'", "level": "INFO", "message": "Starting LR Landing application with RAILS_ENV='"${RAILS_ENV}"'", "RAILS_RELATIVE_URL_ROOT"="'"${RAILS_RELATIVE_URL_ROOT}"'", "SCRIPT_NAME"="'"${SCRIPT_NAME}"'", "API_SERVICE_URL"="'"${API_SERVICE_URL}"'"}'
+echo "{\"ts\": $(date -u +%FT%T.%3NZ), \"level\": \"INFO\", \"message\": \"Starting ${PUBLIC_NAME} application with RAILS_ENV=\"${RAILS_ENV}\", \"RAILS_RELATIVE_URL_ROOT\"=\"${RAILS_RELATIVE_URL_ROOT}\", \"SCRIPT_NAME\"=\"${SCRIPT_NAME}\"}"
```

Comment:
What is `PUBLIC_NAME`?
`RAILS_RELATIVE_URL_ROOT` and `SCRIPT_NAME` should not be used.

Reply:
`PUBLIC_NAME` is set on line 9 (to _LR Landing_) presumably to give one place to change a bit of text that's embedded in two locations.

Not understanding the objection to logging the value of `RAILS_RELATIVE_URL_ROOT` to aid debugging but agreed that `SCRIPT_NAME` is redundant given how it's being set.